lib/models/test-utils/testModelsHelper.js

```diff
@@ -0,0 +1,10 @@
+import User from '../user';
+import { userFixture } from './fixtures/user.fixtures';
+import Organisation from '../organisation';
+import { orgFixture } from './fixtures/organisation.fixtures';
+
+export const createUser = async () =>
+  await new User(userFixture).save();
+
+export const createOrganisation = async () =>
```

Comment:
Yeah, better to pass org fixture here as an argument too

Reply:
Done!